### DISCUSSION TITLE
Bonsai crashes without warning with my workflow

### DISCUSSION THREAD

**Discussion by Enricocloud** [bonsai-rx/bonsai#1506] [Q&A]
Hello, 
after the December release we have been experiencing a lot of crushes when running our bonsai protocol (see the .zip). We have not been able to identify a cause of the crashes and there is no error message coming up on the console. The crashes occur at random, causing the video acquisition to freeze and then the whole program shuts down. We haven't had an issue with other workflows, so most likely it has to do with this specific workflow. Could you maybe take a look at it and suggest possible changes to make it more efficient and stable?

Thank you very much for your time.
[bonsai_protocol.zip](https://github.com/bonsai-rx/bonsai/files/12239556/bonsai_protocol.zip)

**Comment by bruno-f-cruz** [comment]
Hey
Have you tried to systematically disable parts of your workflow to further isolate the bug?
Additionally, have you tried to log the console output (eg #1162)?
Cheers, 
Bruno
